src/components/BarChart.js

```diff
@@ -71,57 +71,60 @@ export class BarChart extends React.Component {
 
   render() {
     return (
-      <div className="chartContainer">
-        <canvas ref={this.canvasRef}>
-          <p>
-            Text alternative for this canvas graphic is in the data table below.
-          </p>
-        <table
-          border="0"
-          cellPadding="5"
-          summary="This is the text alternative for the canvas graphic."
-        >
-          <caption>Records Indexed for Jetpack Search</caption>
-          <tbody>
-            <tr>
-              <th scope="col">Post Type</th>
-              {this.state?.legendItems.length &&
-                this.state.legendItems.map((item) => {
-                  return <th scope="col">{item.text}</th>;
-                })}
-            </tr>
-            <tr>
-              <th scope="row">Record Count</th>
-              {this.state?.legendItems.length &&
+      <div className="barChart">
+        <div className="chartContainer">
+          <canvas ref={this.canvasRef}>
+            <p>
+              Text alternative for this canvas graphic is in the data table below.
+            </p>
+          <table
+            border="0"
+            cellPadding="5"
+            summary="This is the text alternative for the canvas graphic."
+          >
+            <caption>Records Indexed for Jetpack Search</caption>
+            <tbody>
+              <tr>
+                <th scope="col">Post Type</th>
+                {this.state?.legendItems.length &&
+                  this.state.legendItems.map((item) => {
+                    return <th scope="col">{item.text}</th>;
+                  })}
+              </tr>
+              <tr>
+                <th scope="row">Record Count</th>
+                {this.state?.legendItems.length &&
+                  this.state.legendItems.map((item) => {
+                    return (
+                      <td>{this.props.data[item.datasetIndex].data.data}</td>
+                    );
+                  })}
+              </tr>
+            </tbody>
+          </table>
+          </canvas>
+          </div>
+          <div className="chartLegendContainer">
```

Comment:
Created a new container for the legend which is not constrained by the chartContainer styles.